### DISCUSSION TITLE
Auto save in campaign

### DISCUSSION THREAD

**Discussion by sovzzz** [Phobos-developers/Phobos#1072] [Ideas and feedback]
### Description

a map action used in solo campaign maps. When triggered, a game save named Mapname-date-time created. Just similar to "win" action.

### INI code

```ini
;in map file
[Actions]
ActionID=1,ActionCode,0,0,0,0,0,0,A
;in ra2md.ini, enabled with client
[Phobos]
AutoSave = (bool)
```

### Checklist

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this topic.
- [X] I used the GitHub search and read the topic list to find a similar topic and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

**Comment by MortonPL** [comment]
This is already implemented:

https://phobos.readthedocs.io/en/latest/AI-Scripting-and-Mapping.html#save-game
